docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -51,6 +48,14 @@ The first method uses [Amazon EKS Workshop](https://www.eksworkshop.com/010_intr
 
 To manage an Aurora Serverless v2 cluster from Kubernetes / Amazon EKS, you will need to install the ACK for RDS service controller. You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart).
 
+Define environment variables
+
+```
+export SERVICE=rds
+RELEASE_VERSION=$(curl -sL "https://api.github.com/repos/aws-controllers-k8s/${SERVICE}-controller/releases/latest" | grep '"tag_name":' | cut -d'"' -f4)
+export ACK_SYSTEM_NAMESPACE=ack-system
+export AWS_REGION=$(curl -s 169.254.169.254/latest/dynamic/instance-identity/document | jq -r '.region')
```

Comment:
I understand that this tutorial was written to be used within a ec2-based environment, but I am a little hesitant that we recommend using the IMDS URL in tutorials. My preference would be that we ask users which region they would like to deploy resources, and then these scripts can be a little more re-usable for people using them outside Cloud9.

Reply:
do you mean we provide scripts to build EKS inline here? we can try that in the next release?

Reply:
Perhaps to say another way, can we just return to hardcoding the region in? I think users can set the region manually (similar to what we do in the RDS tutorial)